docs/about/contact.md

```diff
@@ -18,6 +18,8 @@ However, the FOSS@RIT identity is not exclusive.
 It is not "owned" by any single group or organization.
 Generally, anyone who works in free/open communities at RIT, not limited to software, is included in this identity.
 
+See Prof. Deejoe's blog post with suggestions on how to get involved with the FOSS@RIT community: [Joining RIT@FOSS](https://web.archive.org/web/20200225161814/http://deejoe.etrumeus.com/update/joining-ritfoss.html)
```

Comment:
Why is this an archive.org url?

Reply:
@kennedy It's a personal preference of mine lately. I like to use links from archive.org since they are better guarantees that a link will still work in 5, 10, 15 years. Probably not necessary here but it was a force of habit for me.